db/tests/types/test_base.py

```diff
@@ -0,0 +1,188 @@
+from datetime import timedelta
+from decimal import Decimal
+import pytest
+from sqlalchemy import Table, Column, MetaData
+from sqlalchemy import String, Numeric
+from sqlalchemy.schema import CreateSchema, DropSchema
+from db import types
+from db.engine import _add_custom_types_to_engine
+from db.types import base, install
+
+TEST_SCHEMA = "test_schema"
+
+
+@pytest.fixture
+def engine_with_types(engine):
+    _add_custom_types_to_engine(engine)
+    return engine
+
+
+@pytest.fixture
+def test_schema(engine_with_types):
```

Comment:
Could you call this something that doesn't begin with `test_`, since function names starting with `test_` are run as tests generally and it has the potential to be confusing.